src/main/resources/templates/about.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Culinary Companion</title>
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="../static/style.css">
+</head>
+<body>
+<div class="success-message" th:if="${session.loginSuccessful}" th:text="${session.loginSuccessful}"></div>
+<header class="header">
+    <nav class="nav-bar">
+        <a href="home" class="logo"><i class='bx bx-dish'></i></a>
+        <a href="home">Home</a>
+        <a href="about">About</a>
+        <a href="contact">Contact</a>
+        <a href="help">Help</a>
+    </nav>
+    <form action="" class="search-bar">
+        <input type="text" placeholder="Search...">
+        <button><i class='bx bx-search-alt-2'></i></button>
+    </form>
+    <a href="userProfile" class="account"><i class='bx bx-user'></i></a>
+</header>
+
+<script src="../static/login.js"></script>
+<script src="../static/error-message.js"></script>
+<script src="../static/success-message.js"></script>
+
+<br>
+<br>
+<br>
+<br>
+<h1>About Us</h1>
+<p>Insert about us information</p>
+<style>
+    h1 {
+        text-align: center;
+        margin-bottom: 10px;
+    }
+
+    p {
+        text-align: center;
+    }
+</style>
+</body>
+</html>
```

Comment:
Adding these missing files will help a lot when moving forward. Use these as templates and add more to the page in the future. Did these changes for the functionality and side effect portion of code reviewing.  
Reference: https://javarevisited.blogspot.com/2011/09/code-review-checklist-best-practice.html#axzz8GGTiv6vY